### PR TITLE
M3 Longview Landing Adjustments

### PR DESCRIPTION
**Pull request by WilkinsKa1**
## Description

This PR ended up being a bit of a mix of visual adjustments mostly to landing but also to plan details (mobile). Of note:

• Removed entity icon from client rows
• Updated location and styling for last updated timestamp (now displays for each client row)
• Packages link styling
• Mobile Landing header
• Mobile Plan Details fix

## Type of Change
- Non breaking change ('update')

### CODE REVIEW

File: packages/manager/src/features/Longview/LongviewLanding/LongviewListRows.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ const LongviewListRows: React.FC<CombinedProps> = props => {
             clientLabel={eachClient.label}
             clientAPIKey={eachClient.api_key}
             triggerDeleteLongviewClient={triggerDeleteLongviewClient}
+            longviewClientLastUpdated={Number(eachClient.updated)}
```
Comment (WilkinsKa1) [inline packages/manager/src/features/Longview/LongviewLanding/LongviewListRows.tsx]: This feels weird/wrong to me- `updated` comes back as a UTC time stamp, but within `LongviewClientHeader` I'm converting `longviewClientLastUpdated`(number) to match a UTC string format and then refining it even further. I did this for now to make the linter happy but wanted to call it out.

Comment (martinmckenna) [reply packages/manager/src/features/Longview/LongviewLanding/LongviewListRows.tsx]: yeah I don't think this prop should even be passed at all - The _last updated a few seconds ago_ text is intended to be generated from the datetime that comes from the Longview API, not APIv4.

In `LongviewClientRow.tsx`, you should pass down the `lastUpdated` from state, and can safely remove the possibility of this being a prop.

This is just a side-effect of you moving the date formatting stuff to the LongviewClientHeader.tsx

Comment (Jskobos) [inline packages/manager/src/features/Longview/LongviewLanding/LongviewListRows.tsx]: This is when the Longview client (the API object) was last updated, which is likely different from when the data (from the Longview API) was last updated. `eachClient.updated` is a string, too, which is what's throwing off your typings here. 

Each client row should already have access to client data, so you can use `longviewClientDataLastUpdated` for display.

**Comment by Jskobos** [comment]
The value that @martinmckenna is talking about is already available in `LongviewClientHeader.tsx` as `longviewClientLastUpdated`. Use that instead of this prop and you'll be good.

**Pull request merged**